vmdb/app/models/orchestration_stack_openstack_infra.rb

```diff
@@ -0,0 +1,13 @@
+class OrchestrationStackOpenstackInfra < OrchestrationStack
+  def raw_update_stack(options)
+    ext_management_system.with_provider_connection(:service => "CloudFormation") do |connection|
```

Comment:
Or leave the two methods unimplemented. I will enable them when we consolidate cloud and infra orchestration refreshes.